render-assets.sh

```diff
@@ -1,17 +1,17 @@
 #!/bin/bash
 set -ueo pipefail
 
-if [[ ! "$(which inkscape 2> /dev/null || which rendersvg 2> /dev/null)" ]]; then
+if [[ ! "$(command -v inkscape || command -v rendersvg)" ]]; then
   echo "'inkscape' or 'resvg' needs to be installed to generate the PNG."
   exit 1
 fi
 
-if [[ ! "$(which optipng 2> /dev/null)" ]]; then
+if [[ ! "$(command -v optipng)" ]]; then
   echo "'optipng' needs to be installed to optimize the resulting PNG!"
 fi
 
 chrome() (
-  cd src/chrome
+  cd ./src/chrome
```

Comment:
Why is `./` necessary only for this? shellcheck does not seem to give warning even without it.

Reply:
i was thinking it could be more readable but it's indeed not important

Reply:
I see. I just wondered why this is only in `src/chrome` but not in `src/gtk` and `src/gtk-2.0` :)